WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderFragment.kt

```diff
@@ -229,6 +221,30 @@ class ReaderFragment : Fragment(R.layout.reader_fragment_layout), MenuProvider,
         viewModel.start()
     }
 
+    private fun ReaderFragmentLayoutBinding.updateUiState(uiState: ReaderViewModel.ReaderUiState?) {
+        uiState?.let {
+            when (it) {
+                is ContentUiState -> {
+                    binding?.readerTopBarComposeView?.apply {
+                        setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
+                        setContent {
+                            AppTheme {
+                                ReaderTopAppBar(onSearchClick = {})
+                            }
+                        }
+                    }
+                    updateTabs(it)
```

Comment:
Good question. I didn't fully investigate the tabs code in the context of this PR since it was just about UI, but we might need some things from this logic (e.g. analytics events if there are any). This is something I'll be revisiting in my current task, which is about replacing the tabs/pager content loading with the drop-down menu. There are probably many things that can be safely removed from this screen.